api/admin/opds.py

```diff
@@ -1,15 +1,17 @@
 from nose.tools import set_trace
 
 from api.opds import CirculationManagerAnnotator
+from core.opds import VerboseAnnotator
 from core.lane import Facets, Pagination
 from core.model import (
     BaseMaterializedWork,
     LicensePool,
     Session,
 )
 from core.opds import AcquisitionFeed
+from core.util.opds_writer import AtomFeed
 
-class AdminAnnotator(CirculationManagerAnnotator):
+class AdminAnnotator(VerboseAnnotator, CirculationManagerAnnotator):
```

Comment:
I found a bug in the complaints and hidden books feeds due to this - they're missing acquisition links.

Reply:
I don't follow -- CirculationManagerAnnotator has the acquisition links but making it also inherit VerboseAnnotator takes them away? Is VerboseAnnotator missing a super() call?

Reply:
Correct, that was the problem. But I need to investigate more, I think there may be something else wrong with those feeds as well.